packer-plugin-sdk/sdk-internals/communicator/ssh/communicator.go

```diff
@@ -1,3 +1,8 @@
+/*
+Package ssh implements the SSH communicator. Plugin maintainers should not
+import this package directly, instead using the tooling in the
+"packer-plugin-sdk/communicator" module.
+*/
```

Comment:
```suggestion
// Package ssh implements the SSH communicator. Plugin maintainers should not
// import this package directly, instead using the tooling in the
// "packer-plugin-sdk/communicator" module.
```

Reply:
By the way on the same topic :  

>  Plugin maintainers should not import this package directly

Should this pkg be made internal ? 

It might not really be possible since a lot of code is accessing the SSH struct.

Reply:
I don't think we can make it internal because of that struct but I can revisit it with fresh eyes now that the split is more complete.